core/lib/libtomcrypt/dsa.c

```diff
@@ -60,41 +60,36 @@ TEE_Result crypto_acipher_alloc_dsa_public_key(struct dsa_public_key *s,
 
 TEE_Result crypto_acipher_gen_dsa_key(struct dsa_keypair *key, size_t key_size)
 {
-	TEE_Result res;
 	dsa_key ltc_tmp_key;
-	size_t group_size, modulus_size = key_size/8;
 	int ltc_res;
```

Comment:
while changing function content, can you also initialize these 2 local variables?

Reply:
Will do